### ISSUE TITLE
Example of @FreeBuilder and @Expose for GSON support

### ISSUE THREAD

**Issue by jaredsburrows** [google/FreeBuilder#195]
The README.md says it does support GSON. Can there be an example?

**Comment by chrisalice** [comment]
Jackson, I'm afraid, not GSON. I assume this is a feature request, though.

**Comment by jaredsburrows** [comment]
I thought this was saying it could support gson: https://github.com/google/FreeBuilder#immutables-vs-freebuilder.

**Comment by chrisalice** [comment]
Immutables supports GSON, FreeBuilder does not.

**Comment by jaredsburrows** [comment]
Then I would like to request this feature.

**Comment by chrisalice** [comment]
Sure thing! Not sure when I'll be able to work on it, but it'll definitely be sooner if you would be willing to peer review the code changes -- would that be okay?

**Comment by jaredsburrows** [comment]
Yeah sure. I just came across this the other day and wanted to check it out. This might be able to support gson out of the box if you simple allow gson to parse all fields without `@Expose`.

**Comment by GeenDutchman** [comment]
I know I'm very late for this party, but is there anything *preventing* the support of GSON?
